templates/digital-asset-template/scripts/move/upgrade.js

```diff
@@ -0,0 +1,35 @@
+require("dotenv").config();
+const cli = require("@aptos-labs/ts-sdk/dist/common/cli/index.js");
+
+async function publish() {
+  if (!process.env.VITE_COLLECTION_CREATOR_ADDRESS) {
+    throw new Error("VITE_COLLECTION_CREATOR_ADDRESS variable is not set, make sure you set it on the .env file");
+  }
+
+  if (!process.env.VITE_MODULE_ADDRESS) {
+    throw new Error(
+      "VITE_MODULE_ADDRESS variable is not set, make sure you have published the module before upgrading it",
+    );
+  }
+
+  const move = new cli.Move();
+
+  move
+    .upgradeObjectPackage({
```

Comment:
this is duplicated with the publish script. can we somehow merge it? maybe for the publish command, call the upgrade command an then run a script to update the .env file?

Reply:
which part is duplicate? you mean the function parameter of `createObjectAndPublishPackage` and `upgradeObjectPackage` is the same?

Reply:
I mean the use of `upgradeObjectPackage` in both publish and upgrade scripts

Reply:
`upgradeObjectPackage` is only used in upgrade script, not in publish script